ea_airflow_util/providers/sharefile/hooks/sharefile.py

```diff
@@ -76,6 +78,43 @@ def download(self, item_id, local_path):
             self.log.error('Item request failed with status code: {}'.format(dl_response.status_code))
             dl_response.raise_for_status()
 
+    def upload_file(self, folder_id, local_file):
+        # establish a session if we don't already have one
+        if not self.session:
+            self.get_conn()
+
+        # referencing sharefile's upload logic here
+        # https://api.sharefile.com/samples/python
+        upload_config_uri = f"{self.base_url}/Items({folder_id})/Upload"
+        upload_config_resp = self.session.get(upload_config_uri)
+        upload_config = upload_config_resp.json()
+    
+        upload_uri = ""
```

Comment:
Are there any cases where `upload_uri` is undefined and it's expected? If not, we can remove line 92.

Reply:
Fair enough. I'd argue that NullPointerExceptions are preferable to using an empty string in compiled languages when alerting that something is mal-defined. Could we set upload_uri to None and leave a comment next to it that this condition will never be met?

Reply:
Happy to set it to `None`